TranscriberModels.py

```diff
@@ -9,22 +9,34 @@ def get_model(use_api):
     else:
         return WhisperTranscriber()
 
-class WhisperTranscriber:
-    def __init__(self):
+class WhisperTranscriber():
+    
+    
+    def __init__(self, language="en"):
+        self.lang = language
         self.audio_model = whisper.load_model(os.path.join(os.getcwd(), 'tiny.en.pt'))
```

Comment:
language not always corresponding to audio_model loaded here, audio model should depend on language